docs/modules/ROOT/pages/ios_faq.adoc

```diff
@@ -33,17 +36,17 @@ https://github.com/owncloud/ios-app/issues/604
 
 Currently this isn't a planned feature. Focus is more to build the best integration in serious 3rd party text editing app.
 
-Here you can find previous discussion:
+Here you can find previous discussions:
 https://github.com/owncloud/ios-app/issues/317
 
-=== Translation missing or bug
+=== Missing translations or translation bugs
 
 You can help translate in your language or fix a bug.
 
 We use Transifex for translations. Please register for an account and join the global community:
-https://www.transifex.com/owncloud-org/owncloud/mobile-ios-app/
+{ios-app-tx-url}
 
 === Testing new features
 
 Join the beta program on iOS:
```

Comment:
Proposing:
`Use {ios-app-beta-url}[this link] to join the beta program on iOS.`

Reply:
fixed with https://github.com/owncloud/ios-app/pull/798/commits/28783c88091138713ac017492fccccda7e21dfd6